feathr_project/setup.py

```diff
@@ -64,8 +64,7 @@
         "click<=8.1.3",
         "py4j<=0.10.9.7",
         "loguru<=0.6.0",
-        "pandas>=1.5.0",
-        "numpy<=1.20.3",  # pin numpy due to pyspark's deprecated np.bool access
+        "pandas",
```

Comment:
Does unpinning numpy work now?
Previously I pinned that due to the conflict w/ pyspark.
Specifically, pyspark's `df.toPandas()` uses `np.bool` which has been deprecated in the newer numpy versions so that throws a NameError exception.

Reply:
I actually removed it from the dependency file since it's not used, so we can save some effort. Users can try to install it in the user workspace